src/sass/_products.scss

```diff
@@ -0,0 +1,259 @@
+  transform: translateX(-50%);
+  width: 100%vw;
+  height: 485px;
+  background: $pink;
+  border-radius: 24px;
+  &::before {
+    content: '';
+    position: absolute;
+    display: block;
+    width: 249px;
+    height: 252px;
+    top: -113px;
+    left: 16px;
+
+    background: url(../images/icecream-mobile.png);
+    z-index: 2;
+  }
+  &--image-vanilla {
+    background: $yellow;
+  }
+  &--image-vanilla::before {
+    content: '';
+    position: absolute;
+    display: block;
+    width: 249px;
+    height: 252px;
+    top: -113px;
+
+    background: url(../images/icecoffee-mobile.png);
+    z-index: 2;
+  }
+  &--image-chocolate {
+    background: $green;
+  }
+
+  &--image-chocolate::before {
+    content: '';
+    position: absolute;
+    display: block;
+    width: 249px;
+    height: 252px;
+    top: -113px;
+
+    background: url(../images/milkshakes-mobile.png);
+    z-index: 2;
+  }
+}
+
+.products__title {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 174px;
+  height: 23px;
+  margin: 161px 53px 66px;
+
+  font-family: $header-font;
+  font-size: ('review-signature', 'standard');
+  font-style: normal;
+  font-weight: 'regular';
+  line-height: 1.15;
+
+  text-align: center;
+  letter-spacing: 'standard';
+  text-transform: uppercase;
+
+  color: $white;
+
+  &::after {
+    content: '';
+    display: flex;
+    position: absolute;
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background: $white;
+    top: 53px;
+    left: 50%;
+    transform: translateX(-50%);
+    box-shadow: 10px 0px 0px $white, -10px 0px 0px $white;
+  }
+
+  &--overlay {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 20px 53px 25px;
+    font-family: $header-font;
+    font-size: ('review-signature', 'standard');
+    font-weight: 'regular';
+    text-transform: uppercase;
+    color: $white;
+    letter-spacing: 0.1em;
+  }
+}
+
+.products__content {
+  margin: 25px 20px;
+  font-family: $standard-font;
+  font-style: normal;
+  font-weight: 'bold';
+  font-size: 'standard';
+  line-height: 1.52;
+
+  text-align: center;
+  letter-spacing: 'standard';
+
+  color: $white;
+}
+
+.products__field {
+  position: relative;
+  display: flex;
+  width: 200px;
+  height: 124px;
+  margin-left: 50%;
+  transform: translateX(-50%);
+
+  font-family: $standard-font;
+  font-style: normal;
+  font-weight: 'bold';
+  font-size: 'standard';
+  line-height: 1.52;
+
+  text-align: center;
+  letter-spacing: 'standard';
+
+  color: $white;
+  margin-bottom: 30px;
+}
+
+.products__button {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  border: none;
+  background-image: url(../images/arrow-right.png);
+  margin-bottom: 40px;
+  margin-left: 120px;
+
+  &:hover,
+  &:focus,
+  &:active {
+    background: $red-1;
+  }
+}
+
+@media screen and (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
```

Comment:
jak narzucasz te wymagania to background image nadpisuje background color i karty znikają